pyqtgraph/functions.py

```diff
@@ -293,75 +343,115 @@ def mkColor(*args):
     return QtGui.QColor(*args)
 
 
-def mkBrush(*args, **kwds):
+def mkBrush(*args, **kargs):
     """
     | Convenience function for constructing Brush.
     | This function always constructs a solid brush and accepts the same arguments as :func:`mkColor() <pyqtgraph.mkColor>`
     | Calling mkBrush(None) returns an invisible brush.
     """
-    if 'color' in kwds:
-        color = kwds['color']
-    elif len(args) == 1:
-        arg = args[0]
-        if arg is None:
-            return QtGui.QBrush(QtCore.Qt.NoBrush)
-        elif isinstance(arg, QtGui.QBrush):
-            return QtGui.QBrush(arg)
+    while ( # unravel single element sublists
+        ( isinstance(args, tuple) or isinstance(args,list) )
```

Comment:
you can do `isinstance(args, (tuple, list))` here

Reply:
👍 done